modules/core/src/v2/environments.ts

```diff
@@ -62,6 +63,7 @@ const mainnetBase: EnvironmentTemplate = {
     'xpub661MyMwAqRbcEtUgu9HF8ai4ipuVKKHBzUqks4jSFypW8dwwQL1zygLgQx99NmC7zJJznSiwKG6RQfVjAKMtCsx8VjR6kQW8x7HrkXFZdnQ',
   hsmXpub:
     'xpub661MyMwAqRbcGEtTFgMAoxMFoqsRdDaiaR63byNfZdV7cBZFvovQSNJ5bpyeoQtuKVgXBk6sFQ7TtvyWqadt41DnKwveYgM5KoU2EKYjdY2',
+  batcherContractAddress: '0xaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa', // TODO replace with mainnet Batcher contract address
```

Comment:
Is there a ticket number we can leave here?

Reply:
I'll make one and add it.